server/service/integration_mdm_test.go

```diff
@@ -12928,9 +12928,32 @@ INSERT INTO host_mdm_apple_declarations (
 		}
 	}
 
+	checkRequestsDatabase := func(t *testing.T, messageType, enrollmentID string, expectedCount int) {
+		mysql.ExecAdhocSQL(t, s.ds, func(q sqlx.ExtContext) error {
+			var output struct {
+				Num int `db:"count"`
+			}
+			if err := sqlx.GetContext(
```

Comment:
You should be able to scan the results directly in an integer variable (no need for the struct wrapper).

Reply:
Done 👍